src/app/App.test.tsx

```diff
@@ -36,7 +39,9 @@ describe('The main UI for Europa...', () => {
   test('contains a working link to the Navigator', async () => {
     const { container } = render(
       <Provider store={createTestStore()}>
-        <App />
```

Comment:
I think there's more in this PR than you intended

Reply:
Now that the fallback code has been merged, I went ahead and rebased this branch to take it out of this PR.